cluster-autoscaler/go.mod

```diff
@@ -22,7 +22,7 @@ require (
 	github.com/satori/go.uuid v1.2.0
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.7.1
-	golang.org/x/crypto v0.0.0-20220112180741-5e0467b6c7ce
+	golang.org/x/crypto v0.0.0-20220314234659-1baeb1ce4c0b
```

Comment:
Do we know that this version has all required patches?

Reply:
yep - that's the version that includes the fix for this particular vuln.

i briefly experimented updating this package to the latest version, but that required all kinds of other changes, including a golang version bump, and then i struggled getting it all to compile and build.